### ISSUE TITLE
Development status of this project

### ISSUE THREAD

**Issue by Ratcreamsoup** [as-ideas/oil#223]
Hi,

One of our advertisers is telling us that they strongly advise us against using OIL for our websites; they specifically claim that OIL does not comply with the current status of the IAB standard and that there has been a severe lack of development in the last months. The last release is now indeed two months old and there hasn't been a lot of commit activity since.

Has AS dropped active development of OIL or are you still fully committed to maintaining an OSS IAB CMP solution? If so, is that claim by our advertising agency just FUD on behalf of their preferred commercial CMP? Is there a roadmap for future OIL releases?

Kind regards

  Markus

**Comment by Waschnick** [comment]
TL;DR:
> Has AS dropped active development of OIL or are you still fully committed to maintaining an OSS IAB CMP solution?
* No! We are still fully commited to this project

> If so, is that claim by our advertising agency just FUD on behalf of their preferred commercial CMP?
* I never heard of this accusation before. As there is no open ticket in this project claiming anything about not complying to the IAB framework: This sounds like false accusation.

> Is there a roadmap for future OIL releases?
* Yes: Custom Vendors (done by Burda), Performance (oil-core.js needs to be smaller), Mobile SDK, more reporting (this is mostly internal for AS), Wordpress/Drupal Plugins, more work on the backoffice (which is not yet open source)

---

Hey Markus,

this is an Open Source project, so anybody is free to contribute. Currently I know Burda (big german publisher) is using OIL an all their websites and they are currently working on a new feature for more customisation of vendors (= custom vendor support). We (as in Axel Springer) work in phases on this project. It depends on what we internally need, and the last 2 month there was not much we needed, so we only did maintenance. I'm quite positive that we will add new features in the beginning of 2019.

And for "2 month no releases" - this is not uncommon. I actually don't understand the concern - why should there be a release more often? Sounds well maintained for any Open Source project to me.

As for not comply with the IAB standard - I never heard about this claim before. AFAIK we are fully complaint, but if you find anything, please lt me know.

What we did in the past month, and what we plan to also release as Open Source, is a backoffice for easier maintenance with multiple websites and creating your own bundles with publishing to a CDN. We currently only use this internally to administrate Axel Springer websites. It's currently not in a state to be used by others, but we aim to work on this further.

About the roadmap: 
* We are currently waiting for any news about a new version of the cookie-string or about the release of the pubvendors - we will adopt new features immediately when they are release within the IAB Framework
* One of our app teams has a mobile SDK on their roadmap
* Drupal and Wordpress plugins are also on top of our backlog


So long
Sebastian

**Comment by Ratcreamsoup** [comment]
Thank you very much for you quick and comprehensive response. I have a very strong feeling, that certain agencies are trying their luck spreading FUD now that the holidays are approaching and everybody is a little bit on edge about what the new year might bring, presumably in order to profit from reselling commercial CMP solutions.

I'm very much for an OSS effort here, not just because of this being free as in beer, but also because I strongly believe that this type of goal is best achieved with a fully transparent aproach which only OSS can provide.

Even though we ourselves cannot contribute code, I have (and will) try my best to provide input with bug reports if we come across them - as I have already done in the past.

So Kudos to you, this is really very much appreciated and does AS a lot of credit within the publisher community!
